common/src/http.rs

```diff
@@ -0,0 +1,29 @@
+use std::time::Duration;
+
+use reqwest::{Response, StatusCode, header};
+
+/// Parse Retry-After header value
+fn parse_retry_after(value: &str) -> Option<Duration> {
+    // Try parsing as seconds (numeric)
```

Comment:
As the spec allows it, we should do it. Maybe it makes sense to have a timestamp rather than a duration. Or have an enum supporting both.